### ISSUE TITLE
Pushing/Pulling behind a proxy

### ISSUE THREAD

**Issue by SamirHafez** [github/VisualStudio#53]
I can actually login behind a proxy unlike #52 but my problems start after that.
As this is something that happens globally with git commands (not just via the GitHub for Visual Studio)  I'm not sure if this belongs here as it might just be my fault for not setting the global proxy in the git configuration.
Having said that, it would be nice to have options for setting git http.proxy and https.proxy flags from the extension.

**Comment by shana** [comment]
Yes, it looks like there's no way inside Visual Studio to set up this information :disappointed: It would definitely be nice to have a way of setting up proxy information, I'll add it to the todo list.

**Comment by SamirHafez** [comment]
Perhaps you could set the proxy globally as you do with the email, something like mapping textboxes to:

`git config --global http.proxy http://proxyuser:proxypwd@proxy.server.com:8080`
`git config --global https.proxy https://proxyuser:proxypwd@proxy.server.com:8080`
